packages/forms/resources/views/components/tabs.blade.php

```diff
@@ -1,4 +1,4 @@
-<div
+<div wire:ignore.self
```

Comment:
Please put this on its own line after `wire:key`. 🙂

Reply:
> Please put this on its own line after `wire:key`. 🙂

it's done

Reply:
Yes, I noticed, thanks!

Reply:
Why this issue appear " Some checks were not successful : fix-code-style / fix-code-style (pull_request) ".

Reply:
Unrelated to this PR.

Reply:
okey,thanks.